client/src/App.js

```diff
@@ -1,13 +1,31 @@
-import './App.css';
-import ShowAllProjects from './components/ShowAllProjects';
-import allProjects from './db';
+import './App.css'
+// import ShowAllProjects from './components/showAllProjects/ShowAllProjects'
+import allProjects from './db'
+import { useState, useEffect } from 'react'
+import AddProject from './pages/AddProject'
+import AllProjects from './pages/AllProjrcts'
+import { Routes, Route } from 'react-router-dom'
+
 
 
 function App() {
+    const [newPr, setNewPr] = useState(allProjects)
 
+    function NewProjectToDb({event, ID, project_name, client, status, employee_name, work_hours, start_date, end_date, sum_hours}){
```

Comment:
`ID` shouldn't be written with a capital letter as in React component names are written with a Capital Letter. I would recommend you to change the name from `ID` to `id`. :) 
And I that there may be a typo and that the name of your function `NewProjectToDb` is maybe `New ProjectToDo`? But it may be not, so I just want to write it here for you to check and change if the name is wrong. 😅